packages/arb-avm-cpp/cmachine/machine.go

```diff
@@ -149,19 +149,19 @@ func (m *Machine) String() string {
 func makeExecutionAssertion(assertion C.RawAssertion) (*protocol.ExecutionAssertion, []value.Value, uint64, error) {
 	sendsRaw := receiveByteSlice(assertion.sends)
 	logsRaw := receiveByteSlice(assertion.logs)
-	debugPrints, err := protocol.BytesArrayToVals(receiveByteSlice(assertion.debugPrints), uint64(assertion.debugPrintCount))
+	debug_prints, err := protocol.BytesArrayToVals(receiveByteSlice(assertion.debug_prints), uint64(assertion.debug_print_count))
```

Comment:
I think `debug_prints` should still be camelCase for golang

Reply:
addressed in later PR